### PR TITLE
Add danseBlockNumber to iele config

### PR DESCRIPTION
**Pull request by LukasGasior1**
https://github.com/input-output-hk/mantis-cardano/pull/3

### CODE REVIEW

File: msg.proto [hunk 1]
```diff
@@ -50,7 +50,7 @@ message EthereumConfig {
 }
 
 message IeleConfig {
-  // empty
+  bytes danseBlockNumber = 1;
```
Comment (loverdos) [inline msg.proto:53]: Do we need to make this optional?

Comment (LukasGasior1) [reply msg.proto:53]: I think not, do we?

Comment (dwightguth) [reply msg.proto:53]: No, you can make it required and always populate it with a value from Mantis's config. Just remember to bump the protobuf version field.

Comment (loverdos) [reply msg.proto:53]: It seems that in protobuf 3 land there are no required fields and everything is optional. If we need to distinguish between the absense of a field and its default value, then we need wrappers.

See also
 * https://github.com/protocolbuffers/protobuf/issues/2497
 * https://github.com/protocolbuffers/protobuf/blob/6a51c03823448a9229c50795ae0d984ac0fe4fbd/src/google/protobuf/wrappers.proto#L31

Bumping our version is of course something we need to do anyway. So `1.2` or `2.0`?

**Review by loverdos — APPROVED** [review]

**Pull request merged**